src/components/SuccessCreationMessage/SuccessCreationMessage.tsx

```diff
@@ -0,0 +1,22 @@
+import { useEffect, useState } from 'react';
+import RegularText from '../Text/RegularText/RegularText';
+import { Container } from './SuccessCreationMessageStyled';
+
+const SuccessCreationMessage = () => {

```

Comment:
Is this used anywhere 🤔

Reply:
Yes, i used in add car form onComplete mutation